### DISCUSSION TITLE
Can't find saved .frpg files

### DISCUSSION THREAD

**Discussion by BekzodIsakov** [nigification/Flowgorithm-Mac-OSX#26] [Q&A]
I'm on Mac m1 and when I save the program in .fprg format, it's not visible inside that folder. But I'm able to open the file through the app.

**Comment by pipinsupipin** [comment]
have you get the answer? can you share here? thanks

**Reply by jostasik** [reply]
Make a note of where you are saving the file! When you're in flowgorithm and the "Save" window appears, make sure that you select the "Z:" hard drive. (That's what your normal macOS hard drive should appear as.) 

You can then traverse into the "Users" folder to access the folders you typically use on your mac everyday. 

Let me know if you have any additional issues! :)
